rslib/src/browser_table.rs

```diff
@@ -21,31 +23,54 @@ use crate::{
     timestamp::{TimestampMillis, TimestampSecs},
 };
 
-#[derive(Serialize_repr, Deserialize_repr, Debug, PartialEq, Clone, Copy)]
-#[repr(u8)]
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone, Copy, Display, EnumIter, EnumString)]
+#[serde(rename_all = "camelCase")]
+#[strum(serialize_all = "camelCase")]
 pub enum Column {
```

Comment:
This is bugging me. We do the exact same things with serde to serialise the columns as sort keys as with strum to (de)serialise them as column keys. But I couldn't get it to work with the old string literals elsewise.
I think the only difference is that the json strings are wrapped in a pair of double quotes.

Reply:
Does this issue go away if we remove the ability to sort via an enum from find_cards()? That functionality is not actually used by Anki (outside of the unit tests) or any add-ons - I added it at one point because I thought it might be nice to have, but I imagine people will just continue to pass the order in as a string instead of going to the trouble of fetching all the columns first, so it's probably not adding any value.

Reply:
No, that's not a problem.
We use strum to
* deserialise the list of active columns received via pb
* deserialise the name of the sort column when a builtin sort order is received via pb
* serialise columns to string keys when all available browser columns are requested via pb

We use serde to
* deserialise the sort column from the config json

So dropping serde is the most realistic. I believe we could achieve that by letting the frontend get the sort column from the config (as it already does) and passing it to the backend explicitly instead of the empty `from_config` variant.

Reply:
Sorry, it can be hard to keep track of all the specifics :-) Passing in the desired value instead of relying on config is probably a better API in any case, but we'd need to be able to distinguish a provided string key from an arbitrary sql order clause that we also accept

Reply:
No problem, I make these pretty lists as much for myself as for you. I certainly didn't have a clear picture of it myself before writing it down. 😅 

If we pass `reverse` as well, we can just use the Builtin overload of find_cards(). Also, I would keep the `order=True` overload because col can simply fetch the config sort itself if it's not provided.